test/FunctionsV1/PlatformSpecificHelpers.FunctionsV1.cs

```diff
@@ -40,6 +40,12 @@ public static class PlatformSpecificHelpers
             var loggerFactory = new LoggerFactory();
             loggerFactory.AddProvider(loggerProvider);
 
+            // Unless otherwise specified, use legacy partition management for tests as it makes them run faster.
```

Comment:
Should we be concerned about using a legacy strategy as the default for our tests? If not, I would prefer if the comments should explain this

Reply:
I'm not super concerned. We test the new strategy logic in the durable task repo, and the main downside to the new strategy is that it can take moderately longer to start up a taskhubworker, and that perf hit adds up over 100s of tests. None of these tests have multiple taskhubworkers (or very few do), so changing the partition management strategy should be irrelevant.

Reply:
Sounds good. I would suggest adding this rationale in the comments themselves, as I could imagine this question coming up again :)